src/app.js

```diff
@@ -1 +1,61 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+console.log('Hallo daar!');
+
+async function fetchData() {
+  try {
+    const result = await axios.get('https://restcountries.com/v2/all');
+    console.log(result.data);
+    return result.data;
+  } catch (e) {
+    console.error(e);
```

Comment:
hier zou je specifieke errors moeten afvangen volgens de opdracht als ik het goed begrepen heb: 
catch (err) {
        console.error(err);
        if (err.response.status === 404) {
            list.replaceChildren();
            errorMessage.textContent = "Page Not Found | 404";
        }
        if (err.response.status === 500) {
            list.replaceChildren();
            errorMessage.textContent = "Internal Server Error | 500";
        }